commons/commons-transpiler/src/main/kotlin/ac/uk/ebi/transpiler/exception/InvalidColumnException.kt

```diff
@@ -0,0 +1,8 @@
+package ac.uk.ebi.transpiler.exception
+
+const val INVALID_COLUMN_ERROR_MSG = "Base columns must match the template"
+
+class InvalidColumnException : RuntimeException() {
```

Comment:
I would include the column that generate the problem in the error message so problems is easier to identify

Reply:
Done